src/main.cpp

```diff
@@ -171,6 +178,9 @@ int main(int argc, char *argv[]) {
 
   boost::shared_ptr<std::ostream> stream { &std::cout, NoDelete {} };
   sink->locked_backend()->add_stream(stream);
+  auto path         = std::filesystem::temp_directory_path();
```

Comment:
What do you think of saving the log next to the config/app files?

Reply:
For Linux temp_directory_path() goes to /tmp I think. This would be better than to ~/.config/sunshine/. Better still would be to make it a config setting so it could be directed to /var/log or whatever the user prefers.

Reply:
Okay, I'm adding the config path for logs, with the sunshine directory being the default one